### PR TITLE
The one with the logout button, links to past problems, and authentication frontend-backend integration in general

### PR DESCRIPTION
**Pull request by paglenn**
still TBD - removing the cookies (we decided this would be done from the backend)- but the logout resets state and redirects the user back to the login component. 
i'll continue refining that and the  sorting , but want the styling folks to have the most current version of what we're working with and backend folks to have all the buttons to feed routes

### CODE REVIEW

**Review by brok3turtl3 — APPROVED** [review]
Nice work Paul!

File: client/src/helpers/methods.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const titleToURL = function (title) {
+  const titleLower = title.toLowerCase();
+  const titleKebab = titleLower.replaceAll(" ", "-");
+  const url = `https://leetcode.com/problems/${titleKebab}/`;
+  return url;
+};
+// async call to LeetCode api - replace with call to our backend when that works.
+async function getLeetCodeProblem(url) {
```
Comment (joie-zhang) [inline client/src/helpers/methods.js:8]: nice!! great choice in improving modularity with centralizing helper methods in a file!

**Review by Ericesposito — APPROVED** [review]
Great work!! I'm glad we have our Log Out functionality up and running!

**Review by joie-zhang — APPROVED** [review]
Wooo!!! Amazing work! Never fails to impress me :D

**Pull request merged**